EosPayload/drivers/position_aware_logging_driver.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from EosPayload.lib.base_drivers.position_aware_driver_base import PositionAwareDriverBase
+
+
+class PositionAwareLoggingDriver(PositionAwareDriverBase):
+
+    def setup(self) -> None:
+        super().setup()
+        self.register_thread('position-logging', self.device_command)
+
+    def device_command(self, logger: logging.Logger) -> None:
+        while True:
+            self.thread_sleep(logger, 10)
+            try:
+                if self.latest_position is not None:
```

Comment:
should we log the position every 10s or log every position update received?  ie override `self.position_callback` instead of making this thread

Reply:
I know PositionAwareDriverBase works on its own from testing, the goal here is to log what a device using the position would get because that behavior is less validated (it also might provide insight if cutdown fails or something)

Reply:
I guess I don't really follow.  This is logging the latest position every 10s, but the latest position is presumably being updated every 1 or 2 secs.  So if the goal is to log what position a PositionAwareDriver would see at a given time, that goal is only being somewhat accomplished

The thing is, this function is really only testing the PositionAwareDriverBase, and this could be accomplished more cheaply.  What I'm suggesting is that instead of making this 'position-logging' dedicated thread, do something like this:

```python
def position_callback(self, _client, _userdata, _message):
  super().position_callback()
  if self.latest_position is not None:
      self.data_log(...)
```

Reply:
Done